lib/StripeObject.php

```diff
@@ -361,7 +361,7 @@ public function serializeParamsValue($value, $original, $unsaved, $force, $key =
         // user intended it to.
         if ($value === null) {
             return "";
-        } elseif (($value instanceof APIResource) && (!$value->saveWithParent)) {
+        } elseif (($value instanceof ApiResource) && (!$value->saveWithParent)) {
```

Comment:
This one seems worrying. Does that mean the instanceof never worked in that `if`?